lsp-types/src/Language/LSP/VFS.hs

```diff
@@ -278,15 +279,6 @@ closeVFS vfs (J.NotificationMessage _ _ params) =
   in (updateVFS (Map.delete (J.toNormalizedUri uri)) vfs,["Closed: " ++ show uri])
 
 -- ---------------------------------------------------------------------
-{-
-
-data TextDocumentContentChangeEvent =
-  TextDocumentContentChangeEvent
-    { _range       :: Maybe Range
-    , _rangeLength :: Maybe Int
-    , _text        :: String
-    } deriving (Read,Show,Eq)
--}
```

Comment:
Why is it is removed? It is included elsewhere or was being unused?

Reply:
It's commented out

Reply:
oh yeah 🤦